MediaBrowser.Model/Net/MimeTypes.cs

```diff
@@ -228,10 +232,10 @@ public static string GetMimeType(string path, bool enableStreamDefault)
             if (StringHelper.EqualsIgnoreCase(ext, ".oga"))
             {
                 return "audio/ogg";
-			}
-			if (StringHelper.EqualsIgnoreCase(ext, ".opus"))
-			{
-				return "audio/ogg";
+      			}
```

Comment:
Please fix these odd spacings.

Reply:
Thats just the fucky way the diff is showing. Download the file and open it, I fixed screwy formatting not created it.